coalib/output/printers/LogPrinter.py

```diff
@@ -1,21 +1,37 @@
+import sys
 import traceback
+import logging
 
 from pyprint.ColorPrinter import ColorPrinter
 
-from coalib.output.printers.LOG_LEVEL import LOG_LEVEL, LOG_LEVEL_COLORS
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
 from coalib.processes.communication.LogMessage import LogMessage
 
 
+class PrinterHandler(logging.StreamHandler):
+    """
+    Handler for logging which writes messages into stdout.
+    """
+
+    def emit(self, record):
+        msg = self.format(record)
+        sys.stdout.write(msg + "\n")
+
+
 class LogPrinter:
     """
+    This class is deprecated and will be soon removed. To get logger use
+    loggin.getLogger('coala.colored') for colored logger and
```

Comment:
How to know if the output is piped somewhere?
IMO, when you're writing something, you know whether it will be piped or no, so you can use what will be the most appropriate for you.